common/src/main/java/com/wynntils/screens/wynntilsmenu/WynntilsMenuScreen.java

```diff
@@ -218,6 +218,38 @@ private static void renderPlayerInfo(PoseStack poseStack, int mouseX, int mouseY
         InventoryScreen.renderEntityInInventoryFollowsMouse(
                 poseStack, posX, posY, 30, posX + 45 - mouseX, posY - 40 - mouseY, McUtils.player());
 
+        if (!Models.Guild.getGuildName().isEmpty()) {
+            String rank = Models.Guild.getGuildRank().getName();
+
+            if (!rank.equals("Recruit")) {
+                rank += " (" + Models.Guild.getGuildRank().getStars() + ")";
+            }
```

Comment:
Extract this to a Models.Guild.getGuildDescription(), so you don't need to expose getStars. Also, you have an enum. Do not get a string from it and do a string comparison! O_o